packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/react-native/installsdk.md

```diff
@@ -32,7 +32,7 @@ github "okta/okta-oidc-ios" ~> 3.5.0
 ```
 Then run `carthage update --platform iOS`.
 
-Open project settings and choose your application target. Then open `Build Phases` and add `OktaOidc.framework` from `ios/Carthage/Build/iOS` into `Embed Frameworks` section
+Open project settings and choose your application target. Then open `Build Phases` and add `OktaOidc.framework` from `ios/Carthage/Build/iOS` into the `Embed Frameworks` section
```

Comment:
can you put a period at the end of this sentence please :)

Open project settings and choose your application target. Then open `Build Phases` and add `OktaOidc.framework` from `ios/Carthage/Build/iOS` into the `Embed Frameworks` section**.**